server/game/core/Constants.ts

```diff
@@ -134,6 +134,8 @@ export enum CardType {
 export enum WildcardCardType {
     Any = 'any',
     NonLeaderUnit = 'nonLeaderUnit',
+    /** Any card type that can be played from hand */
+    Playable = 'playable',
```

Comment:
Would a card played from Smuggle or Discard also count for these?

Reply:
It's really just a shorthand for the list: event, unit, upgrade (excluding tokens). So the type of play card action doesn't come to bear. I can update the comment to be clearer though